src/EditorFeatures/Core.Wpf/InlineHints/InlineParameterNameHintsTaggerProvider.cs

```diff
@@ -7,6 +7,7 @@
 using System;
 using System.ComponentModel.Composition;
 using Microsoft.CodeAnalysis.Editor.Host;
+using Microsoft.CodeAnalysis.Editor.InlineHints;
```

Comment:
Adding namespace that's the same as the one in the file? Either unused or is there a typo somewhere?

Reply:
Fixed.